src/Backend/Core/Engine/TwigTemplate.php

```diff
@@ -148,28 +148,33 @@ private function parseConstants()
             Model::getContainer()->getParameter('site.multilanguage')
         );
 
-        $url = Model::get('url');
+        // check on url object
+        if (class_exists('url')) {
```

Comment:
you need to check if the container has the 'url' service here.
It isn't a class name but a service name
The correct implementation here would be `Model::getContainer()->has('url')`